README.md

```diff
@@ -49,7 +49,19 @@ The media type for JSON Siren is `application/vnd.siren+json`.
       "fields": [
         { "name": "orderNumber", "type": "hidden", "value": "42" },
         { "name": "productCode", "type": "text" },
-        { "name": "quantity", "type": "number" }
+        { "name": "quantity", "type": "number" },
+        { "name": "color", "type": "radio", "title": "Choose a Color", 
+          "value": [
+            { "title": "The Color Red", "value": "red" },
+            { "title": "The Color Green", "value": "green", "selected": true }
+          ]
+        },
+        { "name": "color", "type": "checkbox", "title": "Choose Accessories", 
```

Comment:
Line 59 should be `{ "name": "accessories", "type": "checkbox", "title": "Choose Accessories",`